crates/p2p-stack/src/subactors/kad/client.rs

```diff
@@ -62,6 +63,14 @@ impl Client {
         Ok(ReceiverStream::new(receiver))
     }
 
+    pub async fn remove_record(&self, key: RecordKey) -> RequestResult<(), Void> {
```

Comment:
what about `std::convert::Infallible` instead of `Void`?

Reply:
replaced it